core/src/tools/testfind_fd.cc

```diff
@@ -96,12 +96,12 @@ void ProcessFileset(directordaemon::FilesetResource* director_fileset,
 
   testfind_stats.hard_links = jcr->fd_impl->ff->linkhash->size();
   printf(_("\n"
-           "Total files    : %d\n"
-           "Max file length: %d\n"
-           "Max path length: %d\n"
-           "Files truncated: %d\n"
-           "Paths truncated: %d\n"
-           "Hard links     : %d\n"),
+           "Total files    : %lu\n"
+           "Max file length: %lu\n"
+           "Max path length: %lu\n"
+           "Files truncated: %lu\n"
+           "Paths truncated: %lu\n"
+           "Hard links     : %lu\n"),
```

Comment:
Should we use PRIu64 here ? lu is technically not the standard compliant specifier for uint64_t, but the rest of the code base also just uses %d and %ld everywhere.